File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/WebResponseContextExtensionData.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class WebResponseContextExtensionData{

	@SerializedName("hasDecorated")
	private boolean hasDecorated;

	public void setHasDecorated(boolean hasDecorated){
		this.hasDecorated = hasDecorated;
	}

	public boolean isHasDecorated(){
		return hasDecorated;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/WebResponseContextExtensionData.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class WebResponseContextExtensionData : Parcelable, Serializable {
    @SerializedName("hasDecorated")
    var isHasDecorated = false
    override fun toString(): String {
        return "WebResponseContextExtensionData{" +
                "hasDecorated = '" + isHasDecorated + '\'' +
                "}"
    }
}